public/js/test/fixtures/backbone.pause.onEnter.json

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
There are a couple things I don't love here:
1. having "fake" data with fixtures
2. having a dump of pause data here. 

in some ways both are probably fine, it's just important to see what balance we like for our tests/stories